packages/vite/src/node/plugins/worker.ts

```diff
@@ -7,6 +7,14 @@ import { cleanUrl, injectQuery } from '../utils'
 import Rollup from 'rollup'
 import { ENV_PUBLIC_PATH } from '../constants'
 
+type RequestCredentials = 'omit' | 'same-origin' | 'include'
+type WorkerType = 'classic' | 'module'
+
+interface WorkerOptions {
+  credentials?: RequestCredentials
+  name?: string
+  type?: WorkerType
+}
```

Comment:
nitpick (non-blocking, formatting): Just add a blank line here

```suggestion
}

```